examples/flutter_gallery/lib/demo/fortnightly/README.md

```diff
@@ -0,0 +1,52 @@
+# Fortnightly
+
+A Flutter sample app based on the Material study Fortnightly (a hypothetical, online newspaper.) It
+showcases print-quality, custom typography, Material Theming, and text-heavy UI design and layout.
+
+For info on the Fortnightly Material Study, see: https://material.io/design/material-studies/fortnightly.html
+
+## Goals for this sample
+
+* Help you understand how to customize and layout text.
+* Provide you with example code for
+  * Text
+  * A short app bar (the menu button top left.)
+  * Avatar images
+
+## Widgets / APIs
+
+* BeveledRectangleBorder
+* BoxConstraints on Container
+* CircleAvatar
+* ExactAssetImage
+* Fonts
+* MediaQuery
+* Stack
+* SingleChildScrollView
+* Text
+* TextStyle
+* TextTheme
+
+## Notice
+
+* Theming is passed as a parameter in the constructor of `MaterialApp` (`theme:`).
+* `MediaQuery.padding` returns `EdgeInsets` around notches and virtual home buttons on screens that
```

Comment:
MediaQuery.of(context).padding really

Maybe it doesn't matter.

Reply:
I'll use the correct-er one.